Sources/XMTPiOS/Client.swift

```diff
@@ -602,4 +602,26 @@ public final class Client {
 		}
 		try await client.requestHistorySync()
 	}
+	
+	public func addWallet(account: SigningKey) async throws {
```

Comment:
After adding the wallet to their identity what benefits do they get/ how can the interact with the wallet?

Reply:
This would allow someone else to enter any one of the added wallet addresses and reach the same inbox.

And for an app that has access to any of the added wallets to create a new installation for that inbox.